src/components/Authenticator.tsx

```diff
@@ -0,0 +1,46 @@
+import { Authenticator as AmplifyAuthenticator } from "@aws-amplify/ui-react";
+import { BASE_URL } from "constants/env";
+import { PRIVACY_POLICY, TERMS_OF_USE_NPO } from "constants/urls";
+import ExtLink from "./ExtLink";
+
+export default function Authenticator() {
+  return (
+    <AmplifyAuthenticator
+      components={{
+        SignUp: {
+          FormFields() {
+            return (
+              <>
+                <AmplifyAuthenticator.SignUp.FormFields />
+                <p className="text-sm">
+                  By signing up, you agree to our{" "}
+                  <ExtLink
+                    href={PRIVACY_POLICY}
+                    className="text-blue hover:text-blue-l2"
+                  >
+                    Privacy Policy
+                  </ExtLink>
+                  ,{" "}
+                  <ExtLink
+                    href={`${BASE_URL}/cookie-policy/`}
```

Comment:
nit: should extract cookie policy URL to *constants/urls*, the same way it's done for other URLs

Reply:
this could prob stay in sigup for the meantime just like `/csr-partners` link is just used footer